premake5.lua

```diff
@@ -89,16 +89,17 @@ project "rlExtrasCPP"
 	filter{}
 	
 	location "./rlExtrasCPP/"
-	language "C"
+	language "C++"
+	cppdialect "C++17"
```

Comment:
What specifically needs C++17? I'd like to keep the smallest C++ required.

Reply:
My bad. But shouldn't `rlExtrasCPP` be C++?